webpack.config.js

```diff
@@ -1,38 +1,68 @@
-const HtmlWebPackPlugin = require("html-webpack-plugin");
+const HtmlWebPackPlugin = require('html-webpack-plugin');
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+const webpack = require('webpack');
 
-module.exports = {
+module.exports = (env = {}) => {
+
+  const {mode = 'development'} = env;
+
+  const isProd = mode === 'production';
+  const isDev = mode === 'development';
+
+  const getStyleLoaders = () => {
+    return [
+      isProd? MiniCssExtractPlugin.loader : 'style-loader',
```

Comment:
whitespace is missed. Set up code auto formatter in VS Code

Reply:
installed and fixed